build.gradle

```diff
@@ -184,11 +184,6 @@ def removeTask(tasks, taskName) {
     }
 }
 
-wrapper {
```

Comment:
we don't need this?

Reply:
Technically no, since we ship the wrapper. If you need to generate it you can do:

` ./gradlew wrapper --gradle-version=5.6.3 --distribution-type=all`